README.md

```diff
@@ -52,7 +52,7 @@ git clone https://github.com/adobe-type-tools/afdko.git
 ```
 
 download the latest version of Python 2.7
-[here](https://www.python.org/downloads/) and extract it into the current
+[here](https://www.python.org/ftp/python/2.7.10/python-2.7.10.msi) and extract it into the current
```

Comment:
The MSI file is Windows-only, while Roboto is supposed to be buildable on Linux and Mac too.